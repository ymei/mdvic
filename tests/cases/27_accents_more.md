More accents: $\acute{a}$, $\grave{e}$, $\breve{u}$, $\check{c}$, $\dot{x}$, $\ddot{y}$, $\underline{z}$.

Dots and products: $a \cdot b$, $a \cdotp b$, $f \circ g$, $x \bullet y$.

