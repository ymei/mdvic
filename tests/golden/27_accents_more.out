More accents: á, è, ŭ, č, ẋ, ÿ, z̲.

Dots and products: a · b, a · b, f ∘ g, 
x • y.
